src/Features/CSharp/Portable/ImplementInterface/CSharpImplementExplicitlyCodeRefactoringProvider.cs

```diff
@@ -90,9 +91,11 @@ protected override bool CheckMemberCanBeConverted(ISymbol member)
                 ? identifierName.Parent
                 : identifierName;
 
+            RoslynDebug.Assert(node is object);
```

Comment:
📝 Only necessary because IsMemberBindingExpression is missing a `[NotNullWhen(true)]` attribute

Reply:
This is still necessary even after annotating both those members. The above is an `||` which means it can't guarantee that `IsMemberBindingExpression` is executed hence `identifier.Parent` is a potentiall `null` value. 

---
In reply to: [362655936](https://github.com/dotnet/roslyn/pull/40699#discussion_r362655936) [](ancestors = 362655936)